### ISSUE TITLE
SuperPMI: Investigate MISSING data for Linux/arm collection

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/runtime#80162]
Even though a recent Linux/arm collection seems to be error-free (https://dev.azure.com/dnceng/internal/_build/results?buildId=2076916&view=results), a subsequent replay during the superpmi-diffs pipeline (where, note, the Linux/arm replay is done with on Windows/x86 with a Linux/arm cross-compiler) of a no-diff change shows a significant number of MISSING failues.

Investigate why this is and fix it.

**Comment by BruceForstall** [comment]
It still appears to be the case that even for new Linux/arm collections, a significant number of MISSING data (~3-4%) occurs immediately afterwards.

**Comment by BruceForstall** [comment]
@jakobbotsch Didn't you recently fix this?

**Comment by jakobbotsch** [comment]
Yes, fixed by #112617 (and looks like my issue #104329 was a duplicate of this one)
